<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>717</width>
    <height>549</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>221</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>50</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>221</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>50</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>10</y>
      <width>221</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>50</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>221</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>50</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>160</y>
      <width>221</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>50</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>160</y>
      <width>221</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>50</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>221</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>50</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>310</y>
      <width>221</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>50</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>310</y>
      <width>221</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>50</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>300</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>10</width>
      <height>10</height>
     </size>
    </property>
    <property name="lineWidth">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>681</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>10</width>
      <height>10</height>
     </size>
    </property>
    <property name="lineWidth">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>150</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>10</width>
      <height>10</height>
     </size>
    </property>
    <property name="lineWidth">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_6">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>150</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>10</width>
      <height>10</height>
     </size>
    </property>
    <property name="lineWidth">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>681</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>10</width>
      <height>10</height>
     </size>
    </property>
    <property name="lineWidth">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>0</y>
      <width>20</width>
      <height>441</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>10</width>
      <height>10</height>
     </size>
    </property>
    <property name="lineWidth">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_10">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>310</y>
      <width>20</width>
      <height>141</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>10</width>
      <height>10</height>
     </size>
    </property>
    <property name="lineWidth">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_11">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>170</y>
      <width>20</width>
      <height>141</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>10</width>
      <height>10</height>
     </size>
    </property>
    <property name="lineWidth">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_12">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>10</y>
      <width>20</width>
      <height>441</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>10</width>
      <height>10</height>
     </size>
    </property>
    <property name="lineWidth">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Turn:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>460</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Player 1</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>717</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTicTacToe">
    <property name="title">
     <string>TicTacToe</string>
    </property>
   </widget>
   <addaction name="menuTicTacToe"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
